branding/EULA

```diff
@@ -1,7 +1,8 @@
-The programs included with the XCP-ng system are free software. To know more about licenses of all programs, please read each COPYING file in the /usr/share/doc/* folders.
+The programs included with the XCP-ng system are free software.
+To know more about licenses of all programs,
```

Comment:
I think we should take the opportunity to add "the" before licenses here, shouldn't we?

Reply:
yes :+1: